lib/templates/sharedPart.js

```diff
@@ -69,9 +69,14 @@ class SharedPart {
   }
 
   /** Update template's id for the corresponding firm in template's config file */
-  static updateTemplateId(firmId, name, templateId) {
+  static updateTemplateId(type, envId, name, templateId) {
     let templateConfig = fsUtils.readConfig(this.TEMPLATE_TYPE, name);
-    templateConfig.id[firmId] = templateId;
+    if (type == "firm") {
+      templateConfig.id[envId] = templateId;
+    } else if (type == "partner") {
+      templateConfig.partner_id[envId] = templateId;
+    }
```

Comment:
Considering that we are already creating a `getTemplateId` in `fsUtils` we may want to create a `setTemplateId` (To use in all template types), and also with the `default` statement that throw an "invalid template type error" (that error could be useful to debug if we ever make changes that breaks this `updateTemplateId` calling a wrong type for example :smile: